package.json

```diff
@@ -12,6 +12,8 @@
     "node": ">=20.18.0"
   },
   "scripts": {
+    "build-bigint-buffer": "[ -d node_modules/bigint-buffer ] && command -v node-gyp > /dev/null && cd node_modules/bigint-buffer && node-gyp configure && node-gyp build || echo 'Skipping bigint-buffer build: folder or node-gyp not found'",
```

Comment:
Same as [in relayer](https://github.com/across-protocol/relayer/blob/b082eb2150a4ae1bc3619eca98a8821ebfa6ed30/package.json#L52-L53) to get rid of the bigint-buffer warning